packages/evm/src/opcodes/util.ts

```diff
@@ -271,3 +272,24 @@ export function updateSstoreGas(
     return common.param('sstoreSetGas')
   }
 }
+
+export const getImmediate = (opcode: number, code: Uint8Array, pc: number) => {
+  let immediate: Uint8Array
```

Comment:
Yes, for this, use the `eof/stackdelta.ts` and then the `immediate` value. The only exception is RJUMPV, which is thus dynamic. (Immediates depend upon the table length)